app/views/foods/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>FOOD Search Results: </h1>
```

Comment:
This value of all of the results is returned as a `totalHits` key in your json response.